salt/modules/lxd.py

```diff
@@ -3606,43 +3603,3 @@ def _pylxd_model_to_dict(obj):
         if hasattr(obj, key):
             marshalled[key] = getattr(obj, key)
     return marshalled
-
-
-#
-# Monkey patching for missing functionality in pylxd
-#
-
-if HAS_PYLXD:
```

Comment:
How old is this version that this patch was used for?  Would it make more sense to only use this patch for the version it works with, this way it still stays functional for users on an older version.

Reply:
This module requires PyLXD >= 2.2.5 which already doesn't require this patch anymore, so removing it doesn't break anything as far as I can see. It was just forgotten and never removed.